Gemfile.jruby-1.9.lock

```diff
@@ -8,14 +8,16 @@ PATH
       filesize (= 0.0.4)
```

Comment:
plus, our version constrains ***should*** be good to allow acceptable dependencies upgrades